controller/backing_image_manager_controller.go

```diff
@@ -649,12 +643,20 @@ func (c *BackingImageManagerController) prepareBackingImageFiles(currentBIM *lon
 			}
 			continue
 		}
+
 		bids, err := c.ds.GetBackingImageDataSource(biName)
 		if err != nil {
 			log.WithError(err).Warn("Failed to get backing image data source before preparing files, will skip handling this backing image")
 			continue
 		}
 
+		currentInfo, exists := currentBIM.Status.BackingImageFileMap[biName]
+		requireFile := !exists || currentInfo.State == longhorn.BackingImageStateFailed
+		requireFile = requireFile || bids.Status.CurrentState == longhorn.BackingImageStateFailed
```

Comment:
NIT: As mentioned above, this is confusing without the explanation in [this comment](https://github.com/longhorn/longhorn/issues/6086#issuecomment-1590662594)

Reply:
@WebberHuang1118 you should resolve the comment until you fix the comment.

Reply:
Got it, thanks